contracts/src/interfaces/IRethHyperdriveRead.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { IHyperdriveRead } from "./IHyperdriveRead.sol";
+import { IRocketStorage } from "./IRocketStorage.sol";
+import { IRocketTokenRETH } from "./IRocketTokenRETH.sol";
+
+interface IRethHyperdriveRead is IHyperdriveRead {
```

Comment:
We should add a `IRethHyperdrive` interface that consumes this if we don't have one already. Also, looking at this again, I'm wondering if `RETH` would be more consistent in these names than `Reth`. We should consider changing this everywhere.